### ISSUE TITLE
Allow cut/paste in terminal with CTRL-SHIFT-C / CTRL-SHIFT-V

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#1687]
From https://community.rstudio.com/t/allow-cut-paste-in-terminal-with-ctrl-shift-c-ctrl-shift-v

while working in the RStudio bash terminal on Linux, it seems that it is not possible to use the “standard” cut and paste shortcuts (CTRL-SHIFT-C / CTRL-SHIFT-V). Is there a way to “enable” this ?

If not, I would consider it a useful improvement for future versions

**Comment by hzarkoob** [comment]
A related point: On Windows **FireFox** 62.0.3, the only _paste_ option in the RStudio built-in terminal seems to be `SHIFT+INSET`. 

PS. I checked and verified the following options do not work:
- `CTRL+V`
- Right click and then select **Paste**

### Systems' details

    RStudio Edition   :  Server
    RStudio Version   :  1.1.423
    Local OS Version  :  Windows 7
    Remote OS Version :  Linux
    R Version         :  3.5.1
    Browser           :  FireFox 62.0.3

**Comment by iago-pssjd** [comment]
Not sure if related but, if I copy -with control+C or with edit>copy- something in RStudio, 
- in Console or a View tab and

1. I paste in firefox address bar, it works.
2. I paste in body of online outlook email it does not work (but in gmail it does)

- in source, then "paste" works always.

Linux 4.9.0.12
Debian 4.9.210-1
RStudio 1.2.5042
Firefox 68.7.0esr

**Comment by liar666** [comment]
Just installed rstudio-1.4.1717-amd64.deb . Now copy/paste "a la" Linux does not work at all in the RStudio console.

This is a total no-go for me: if software is not able to respect the well thought / easy-to-work-with / coherent / etc. Linux UX, then I'll not use it at all. Wasting my time adapting to the Gamer's (M$) or Designers (:apple:) unusable way of doing things (UX) is just not an option.

**Comment by ronblum** [comment]
@liar666 I'm sorry it's not working! We certainly should have some method of cut-and-paste, although we have browser limitation to contend with.
* Do you mean that even right-clicking isn't working, or just shortcuts?
* What was the last version that this worked for you?
* Can you provide your system details, so that I can try to reproduce?

### System details

    RStudio Edition :
    RStudio Version : 
    OS Version      :  
    R Version       :  
    Browser         :

**Comment by consumere** [comment]
RStudio 2021.09.0+351 "Ghost Orchid" Release (077589bcad3467ae79f318afe8641a1899a51606, 2021-09-20) for Windows
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) QtWebEngine/5.12.8 Chrome/69.0.3497.128 Safari/537.36

I use wsl bash inside Rstudio.
Currently, right click for  copy /paste is the only way to interact between the source panel and the terminal panel inside Rstudio. 
It would be great if there was an option to >>enable shortcuts << 
in Global Options -->Terminal Options

**Comment by ronblum** [comment]
@consumere On Windows, do the following work?
* Copy: <kbd>Ctrl-Insert</kbd>
* Paste: <kbd>Shift-Insert</kbd> or <kbd>Ctrl-Shift-V</kbd>

**Comment by consumere** [comment]
@ronblum 
this works, if i use the windows on-screen keyboard.
my problem is, i'am using this setup via remote-desktop so it would be nice if the copy and paste commands were editable.

**Comment by ronblum** [comment]
@consumere Makes sense! Thank you for including the suggestion, as a possible way to address this.

**Comment by francisbarton** [comment]
In RStudio Spotted Wakerobin on Windows 10, I can't paste into the Terminal with <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>V</kbd>.
I can use the mouse right-click menu or <kbd>Shift</kbd>+<kbd>Insert</kbd> successfully. But <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>V</kbd> is in my muscle memory!
I am using Git Bash in the terminal.

**Comment by francisbarton** [comment]
@ronblum Ctrl+Shift+V is working for paste in Git Bash in 2023.12.0+369, but Ctrl+Shift+C _doesn't_ work! I was just about hoping this one could be closed at last...!
